src/plugins/playlist/utils/dom.js

```diff
@@ -0,0 +1,8 @@
+const wrap = (el, wrapper) => {
+  el.parentNode.insertBefore(wrapper, el);
+  wrapper.appendChild(el);
```

Comment:
I know that this is an old function: but does not the code eventually leaves duplicate el one that is wrapped and one that is not wrapped? I mean this seems intended but i find it a bit weird, i thought this function would be deleting the "unwrapped element"?

Reply:
No, it inserts `wrapper` before `el` and then moves `el` into `wrapper`

Reply:
@tsi but it does not remove el from its parent right?